### PR TITLE
allow zmon set opsgenie alert custom fields

### LINKED ISSUE #357: allow setting custom fields when creating opsgenie alert
when creating an opsgenie alert, it's required to be able to optionally set custom fields in the alert.
in such case notify_opsgenie should look like:

`notify_opsgenie(teams=['zmon', 'ops'], message='Number of failed requests is too high!', include_alert=True,customfields={"runbooks":"https://runbo.stups.zalan.do/teams/zerowing/runbooks/marid_rollback"})`

### PR DESCRIPTION
**Pull request by momonem**
resolves #357 

- added parameter for setting custom fields
- touched the tests to include the new parameter

### CODE REVIEW

File: zmon_worker_monitor/zmon_worker/notifications/opsgenie.py [hunk 1]
```diff
@@ -34,6 +32,7 @@ def notify(cls,
                priority=None,
                message='',
                description='',
+               customfields=None,
```
Comment (vetinari) [inline zmon_worker_monitor/zmon_worker/notifications/opsgenie.py]: can we name it `custom_fields`? :)

Comment (momonem) [reply zmon_worker_monitor/zmon_worker/notifications/opsgenie.py]: done, i was just trying to follow opsgenie naming convention but it's better that way :)

**Comment by vetinari** [comment]
:+1:

**Comment by femueller** [comment]
👍

**Pull request merged**